src/components/App/index.jsx

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
-import { BrowserRouter, Link, Route } from 'react-router-dom';
+import { BrowserRouter, Link, Route, Switch } from 'react-router-dom';
 import Home from '../Home';
 import Appointments from '../Appointments';
 import Register from '../Register';
 import SignIn from '../SignIn';
 import Request from '../Request';
 import Availability from '../AdminAvailability';
+import Christianviews from '../Request/Christianview';
```

Comment:
These files are essentially the same. Condense the three components into one and pass down differences through props